### DISCUSSION TITLE
Multiple people working on the same space in Spoke

### DISCUSSION THREAD

**Discussion by emclaren** [mozilla/hubs#2720] [General Questions & Discussion] [answered]
If I may ask some general advice questions:  I'm working with a team to create a virtual gallery in Hubs.  We're trying to figure out the best team workflow because numerous people will be working on the same space.  One issue is how to share assets between us.  

Some recommend getting a Sketchfab pro account so that our asset creators can upload assets privately for the rest of us to use.  My question is:  Is it true that in doing this method, we can skip the baking process?  Saw a youtube video in which this seems to be the case.  If we just zip up our materials and upload them to Sketchfab along with the asset, will we be able to then grab the URL and paste it into our scene like usual and still retain the textures?

Note: this question was copied over from Discord

**Comment by misslivirose** [comment, marked as answer]
There are a few points here that relate to the overall question. Right now, we don't have a multi-user way to edit in Spoke using different accounts. One option that we've seen other groups use and occasionally use ourselves on our team is a shared sign-in with a password in a password manager. This allows people to work on a shared _project_, but not simultaneously. Changes will overwrite other work that people could be doing. 

As far as sharing assets goes, if the asset is in a private account on Sketchfab, they will not resolve in Hubs or Spoke. The model URLs need to be publicly accessible on the web so that the Spoke and Hubs clients are able to fetch them without additional authorization. 

The way that we fetch assets from Sketchfab is to request the .glb version of the file that is uploaded there. I believe that Sketchfab has an auto-conversion process that allows uploaded models to be downloaded in the glTF format. However, if the original model format is different (for example, an FBX model), and that format supports texture or material characteristics that the glTF format doesn't, you may experience issues with auto-conversion and having your models appear as intended. To my knowledge, Sketchfab does not automatically handle lightbaking, though this step is not required for models to work properly. Scenes that rely on dynamic lighting will appear unlit on mobile devices, but materials and textures will still be present.

**Reply by hilarytsai** [reply]
Thank you, @misslivirose!  This is very helpful to know.  So in terms of sharing assets between a team, would you actually recommend Sketchfab for that? We can make it a public account--I don't think anyone's adverse to that.  But if there is a more common way that is considered best practice for teams to create custom assets and make sure others who work on the same Spoke account have access to them, please let me know.  I only mentioned Sketchfab because of Ben Kreimer's video on youtube about uploading assets there for use in Spoke (and auto-conversion being convenient, though most of us I think have a gltf exporter) Thanks again!!!

edit: I'm not sure if the answer was edited after I posted this, but wow I just realized you answered my question about sketchfab and somehow my brain went right back to asking the same thing. I'm so sorry for that! Totally understand where you're coming from now.

**Comment by gfelice** [comment]
Any updates on a multi-user login system?  I'm working with teams of students that are building a hub as a group.  It would be ideal if multiple users could access the same hub.  If not, I saw the suggestion for a password sharing system, but currently hubs does not allow me to use a password.  Everytime I login, it sends me a link to login which is really frustrating.  Is there another option that I'm missing?
